### ISSUE TITLE
In-app notifications should be read by the screen reader on macOS and iOS

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#8836]
Currently, in-app notifications are read by the screen reader only on Windows. [https://mozilla-hub.atlassian.net/browse/VPN-5489](https://mozilla-hub.atlassian.net/browse/VPN-5489|smart-link) has information about how this can be implemented on macOS and iOS.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-6035)
┆Reporter: Vinoo Cherian
